client/src/components/pages/RespondentInfo.js

```diff
@@ -56,21 +56,19 @@ const RespondentInfo = props => {
                 school,
                 dateOfBirth,
                 email,
-                hashKey
+                quizzes: [{
```

Comment:
is there a reason hashkey needs to be in quizzes? can't we have one hashkey per user since quiz id is going to be unique  for the link?